dev/server/backend/database/init/1_insert.sql

```diff
@@ -331,3 +331,48 @@ BEGIN
 END
 $$;
 
+-- game_itemテーブルにダミーデータを挿入
+INSERT INTO game_item (item_id, item_type, atk, hit_rate, crit_dmg, crit_rate) VALUES
+('00000000-0000-0000-0000-000000000001', '武器', 120, 0.85, 60, 0.25),
+('00000000-0000-0000-0000-000000000002', '武器', 200, 0.60, 90, 0.40),
+('00000000-0000-0000-0000-000000000003', '支援アイテム', 30, 1.00, 0, 0.10),
+('00000000-0000-0000-0000-000000000004', '支援アイテム', 0, 0.00, 50, 0.00);
+
+-- game_progressテーブルにダミーデータを挿入
+CREATE OR REPLACE FUNCTION update_progress_percentage()
+RETURNS TRIGGER AS $$
+DECLARE
+    total_stages INTEGER := 3; -- 総ステージ数
+BEGIN
+    -- 小数→四捨五入して整数にして保存
+    NEW.progress_percentage := ROUND((NEW.cleared_stages::NUMERIC / total_stages) * 100);
+    RETURN NEW;
+END;
+$$ LANGUAGE plpgsql;
+CREATE TRIGGER trg_update_progress_percentage
+BEFORE INSERT OR UPDATE ON game_progress
+FOR EACH ROW
+EXECUTE FUNCTION update_progress_percentage();
+INSERT INTO game_progress (user_id, cleared_stages, updated_at) VALUES
+('11111111-1111-1111-1111-111111111111', 1,  '2025-06-10 12:30:00'),  -- → 33%
+('22222222-2222-2222-2222-222222222222', 2, '2025-06-12 15:45:00'),  -- → 67%
+('33333333-3333-3333-3333-333333333333', 3, '2025-06-14 09:00:00'),  -- → 100%
+('44444444-4444-4444-4444-444444444444', 0,  '2025-06-01 08:00:00');  -- → 0%
+SELECT * FROM game_progress;
+
+-- game_logsテーブルにダミーデータを挿入
+INSERT INTO game_logs (user_id, log_type_id, details, created_at) VALUES
+-- ログタイプ: 1 = アイテム使用
+('11111111-1111-1111-1111-111111111111', 1,
+ '{"item_id": "item001", "item_name": "回復薬", "effect": "HP+50"}',
```

Comment:
```suggestion
 '{"item_id": "item001", "item_name": "回復薬", "effect": "HP+50"}',
```

detailsのデータ形式ですが、基本設計仕様書のものも試して頂きたいです。
お手数ですがよろしくお願いします。

<img width="497" alt="スクリーンショット 2025-06-27 9 16 01" src="https://github.com/user-attachments/assets/91bcaba4-cbd8-4c51-bfd7-09e011eb3cc5" />

Reply:
解決済みです。